ArduPlane/servos.cpp

```diff
@@ -440,6 +440,15 @@ void ParametersG2::FWD_BATT_CMP::update()
 // Apply throttle scale to min and max limits
 void ParametersG2::FWD_BATT_CMP::apply_min_max(int8_t &min_throttle, int8_t &max_throttle) const
 {
+    // cut off throttle if FWD_BAT_IDX battery is below FWD_THR_CUTOFF_V resting voltage
+    if (is_positive(batt_voltage_throttle_cutoff) &&
```

Comment:
does this apply in manual throttle modes, such as FBWA? It would be nice to be able to switch to FBWA to recover. Maybe check the auto-throttle flag in the mode?

Reply:
maybe also check healthy() on the battery index?

Reply:
FBWA will return true for that case `throttle_passthru_stabilize` is 0 by default. So throttle limits will still apply. You can use `plane.control_mode->does_auto_throttle()`

Reply:
I think its fine. The idea being in manual throttle mode its up to the pilot to decide what to do. In this case you could well be in the "very low battery" code path that turns off the normal battery compensation in anycase, https://github.com/ArduPilot/ardupilot/blob/09c8ad81bb1624fb049967ab2987dc0a2aafd0b0/ArduPlane/servos.cpp#L419-L422